en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-kubernetes-helm-artifacts.md

```diff
@@ -0,0 +1,193 @@
+
+2.  Use the following guidelines to update the deployment configurations:
+
+    -   **Updating the WSO2 subscription details**
+
+        You can update the username and password in the following section. If you don't have an active WSO2 subscription, leave these parameters empty.
+
+        ```yaml
+        wso2:
+            subscription:
+                username: "<username>"
+                password: "<password>"
+        ```
+
+        Alternatively, you can skip this step and pass your subscription details at the time of deploying (see the next step for details).
+
+    -   You can update [other configurations](https://github.com/wso2/kubernetes-microgateway/tree/v1.0.0.1/helm/choreo-connect/README.md) as required.
+
+3.  Save the `values.yaml` file.
+
+### Step 3 - Deploy Choreo Connect
+
+Once you have set up your Helm resources locally, follow the instructions given below to set up the deployment.
+
+1.  Open a terminal and navigate to the `<KUBERNETES_HOME>/helm/choreo-connect` directory.
+2.  Execute the command that is relevant to your Helm version.
+
+    !!! Tip
+        Be sure to replace `NAMESPACE` with the Kubernetes namespace in which your resources are deployed.
+
+    -   Using **Helm v2**
+
+        ```bash
+        helm install --name <RELEASE_NAME> wso2/choreo-connect --version 1.0.0-1 --namespace <NAMESPACE>
+        ```
+
+    -   Using **Helm v3**
+
+        ```bash
+        helm install <RELEASE_NAME> wso2/choreo-connect --version 1.0.0-1 --namespace <NAMESPACE> --create-namespace
+        ```
+
+#### Update configurations during deployment
+
+If required, you can set any of the deployment configurations at the time of running the deployment (instead of
+specifying them in the `values.yaml` file). See the examples given below.
+
+-   Setting the subscription username and password.
+
+    ```bash
+    --set wso2.subscription.username=<SUBSCRIPTION_USERNAME>
+    --set wso2.subscription.username=<SUBSCRIPTION_USERNAME>
+    ```
+
+## Access the Choreo Connect deployment
+
+Follow the steps given below.
+
+1.  Get the external IP (`EXTERNAL-IP`) of the Ingress resources by listing down the Kubernetes Ingresses.
+
+    ```bash
+    kubectl get ing -n <NAMESPACE>
+    ```
+
+    Example:
+
+    ```bash
+    NAME                                       HOSTS                      ADDRESS        PORTS     AGE
+    <RELEASE_NAME>-choreo-connect-adapter      adapter.wso2.com           <EXTERNAL-IP>  80, 443   3m
+    <RELEASE_NAME>-choreo-connect-router       gw.wso2.com                <EXTERNAL-IP>  80, 443   3m
+    ```
+
+2.  Add the above hosts in the `/etc/hosts` file as follows:
+
+    ```bash
+    <EXTERNAL-IP>   adapter.wso2.com
+    <EXTERNAL-IP>   gw.wso2.com
+    ```
+
+3.  Execute the following command to invoke health check services:
+
+    ```bash
+    curl -X GET "https://gw.wso2.com/health"
+    ```
```

Comment:
Let's remove this step 3.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/5214/commits/56e8d06f0eefdc6ceb6870305d7af0214650c154